examples/vision/video_classification.py

```diff
@@ -1,19 +1,25 @@
 """
 Title: Video Classification with a CNN-RNN Architecture
-Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Author: [Sayak Paul](https://twitter.com/RisingSayak), [Mikolaj Buchwald](https://www.linkedin.com/in/mikolaj-buchwald/)
 Date created: 2021/05/28
-Last modified: 2021/06/05
+Last modified: 2023/05/12
 Description: Training a video classifier with transfer learning and a recurrent model on the UCF101 dataset.
 Accelerator: GPU
 """
 """
 This example demonstrates video classification, an important use-case with
 applications in recommendations, security, and so on.
+
 We will be using the [UCF101 dataset](https://www.crcv.ucf.edu/data/UCF101.php)
 to build our video classifier. The dataset consists of videos categorized into different
 actions, like cricket shot, punching, biking, etc. This dataset is commonly used to
 build action recognizers, which are an application of video classification.
 
+The subset (subsampling) of the UCF-101 dataset that we will be working on is available
+at: https://zenodo.org/record/7924745#.ZFztKy1Bxqs (DOI:
+[10.5281/zenodo.7924745](https://doi.org/10.5281/zenodo.7924745)) and will be downloaded
+using the [tf.keras.utils.get_file](https://www.tensorflow.org/api_docs/python/tf/keras/utils/get_file) function.
+
```

Comment:
Is this the same subset that was used previously in the example?

Reply:
No, it is subset reduced to top5 movie categories from the UCF101, because when I used the notebook that is mentioned to generate a subset of top10 videos, there were twice as many videos as used previously in the `video_classification.py` example, and due to that `autogen.py`/`tutobooks.py` timeout of 60*60 seconds (1h) was exceeded.

Reply:
> because when I used the notebook that is mentioned to generate a subset of top10 videos, there were twice as many videos as used previously in the video_classification.py

So, currently, if we use the Top10 subset the current runtime errors out. Is that the problem?

Reply:
Yes, because if we will use Top10 now, then during generation of `ipynb` and `md` files with the `autogen.py` script, there will be timeout errors (due to the `60*60` timeout setting in `tutobooks.py` script). (For the notebook itself it is not the problem but I guess that for the workflow for generating the docs for keras.io it can be a problem). A workaround, if we still very much would like to use the Top10 subset, is to temporarily change the timeout in `tutobooks.py` file for, e.g. `60*600` -- then the generation will pass without errors. I would not recommend this option, because if the example would be to be modified in the future, someone may not know/don't remember about he timeout problem with Top10 subset.